website/content/en/docs/tasks/deprov-nodes.md

```diff
@@ -1,36 +1,84 @@
 ---
-title: "Deprovisioning nodes"
-linkTitle: "Deprovisioning nodes"
-weight: 20
+title: "Deprovisioning"
+linkTitle: "Deprovisioning"
+weight: 10
 ---
 
+Karpenter sets a Kubernetes [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) on each node it provisions.
+The finalizer specifies additional actions the Karpenter controller will takes in response to a node deletion request.
+These include:
 
-## Deletion Workflow
+* Marking the node as unschedulable, so no further pods can be deployed there.
+* Evicting all workload pods from the node, draining it of all pods other than daemonsets.
+* Deleting the node from the Kubernetes cluster.
+* Terminating the instance from the cloud provider.
 
-### Finalizer
+## How Karpenter nodes are deprovisioned
 
-Karpenter adds a finalizer to provisioned nodes. [Review how finalizers work.](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/#how-finalizers-work) 
+There are both automated and manual ways of deprovisioning nodes provisioned by Karpenter:
 
-### Drain Nodes
+* **Node empty**: Karpenter notes when the last workload (non-daemonset) pod stops running on a node. From that point, Karpenter waits the number of seconds set by `ttlSecondsAfterEmpty`  in the provisioner, then Karpenter requests to delete the node. This feature can keep costs down by removing nodes that are no longer being used for workloads.
+* **Node expired**: Karpenter asks to delete the node after a set number of seconds, based on the provisioner `ttlSecondsUntilExpired`  value, from the time the node was provisioned. One use case for node expiry is to handle node upgrades. Old nodes (with a potentially outdated Kubernetes version or operating system) are deleted, and replaced with nodes on the current version.
```

Comment:
We should keep the verbiage the same as used above. 
```suggestion
* **Node expired**: Karpenter requests to delete the node after a set number of seconds, based on the provisioner `ttlSecondsUntilExpired` value, from the time the node was provisioned. One use case for node expiry is to handle node upgrades. Old nodes (with a potentially outdated Kubernetes version or operating system) are deleted, and replaced with nodes on the current version.
```

Reply:
> and replaced with nodes on the current version.

This assumes that the user has configured their resources to use the latest version rather than a specific version. This may be too in the weeds, but I think it's good to note that this feature doesn't automatically version nodes.